catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/source/WfsResponseExceptionMapperTest.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.v1_1_0.catalog.source;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.io.ByteArrayInputStream;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import org.codice.ddf.spatial.ogc.wfs.catalog.common.WfsException;
+import org.junit.Test;
+
+public class WfsResponseExceptionMapperTest {
+
+  @Test
+  public void testCswExceptionWithNullResponse() {
+    WfsException cswException = new WfsResponseExceptionMapper().fromResponse(null);
+
+    assertThat(cswException.getMessage(), equalTo("Error handling response, response is null"));
```

Comment:
String comparison on the exception message seems a little brittle

Reply:
I'll work on subclassing WfsException so that we can do an instanceof check instead of a string check